presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -189,6 +268,10 @@ public void testLeftTrim()
         assertFunction("LTRIM('  hello')", VARCHAR, "hello");
         assertFunction("LTRIM('hello  ')", VARCHAR, "hello  ");
         assertFunction("LTRIM(' hello world ')", VARCHAR, "hello world ");
+
+        assertFunction("LTRIM('\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ')", VARCHAR, "\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ");
+        assertFunction("LTRIM(' \u4FE1\u5FF5 \u7231 \u5E0C\u671B ')", VARCHAR, "\u4FE1\u5FF5 \u7231 \u5E0C\u671B ");
+        assertFunction("LTRIM('  \u4FE1\u5FF5 \u7231 \u5E0C\u671B')", VARCHAR, "\u4FE1\u5FF5 \u7231 \u5E0C\u671B");
```

Comment:
Add tests for unicode spaces, and ascii non-32 spaces. Same for `RTRIM`, `TRIM`.